libcudacxx/include/cuda/std/__atomic/operations/host.h

```diff
@@ -0,0 +1,189 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _LIBCUDACXX___ATOMICS_HOST_H
+#define _LIBCUDACXX___ATOMICS_HOST_H
```

Comment:
We conventionally use the full name as the header guard so:
```suggestion
#ifndef _LIBCUDACXX___ATOMIC_OPERATIONS_HOST_H
#define _LIBCUDACXX___ATOMIC_OPERATIONS_HOST_H
```